unit10/unit10.js

```diff
@@ -0,0 +1,145 @@
+//Bai : 1  in một bảng của một số bất kì với các số từ 1 đến 10 và hiển thị ra kết quả 
+const tableMultiplication = (number) => {
+  if(number >= 1 && number <= 10) {
```

Comment:
```suggestion
  if (number >= 1 && number <= 10) {
```
Nhìn kỹ cách viết dấu cách